_posts/en/newsletters/2024-10-25-newsletter.md

```diff
@@ -0,0 +1,174 @@
+---
+title: 'Bitcoin Optech Newsletter #326'
+permalink: /en/newsletters/2024/10/25/
+name: 2024-10-25-newsletter
+slug: 2024-10-25-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Updates to the version 1.75 channel announcements proposal:** Elle
+  Mouton [posted][mouton chanann] to Delving Bitcoin a description of
+  several proposed changes to the [new channel announcements][topic
+  channel announcements] protocol that will support advertising [simple
+  taproot channels][topic simple taproot channels].  The most
+  significant planned change is to allow the messages to also announce
+  current-style P2WSH channels; this will allow nodes to later "start
+  switching off the legacy protocol [...] when most of the network seems
+  to have upgraded".
+
+  Another addition, recently discussed (see [Newsletter #325][news325
+  chanann]), is to allow announcements to include an SPV proof so that
+  any client that has all of the headers from the most-proof-of-work
+  blockchain can verify that the channel's funding transaction was
+  included in a block.  Currently, lightweight clients must download an
+  entire block to perform the same level of verification of a channel
+  announcement.
+
+  Mouton's post also briefly discusses allowing opt-in announcement of
+  existing simple taproot channels.  Due to the current lack of support
+  for announcements of non-P2WSH channels, all existing taproot channels
+  are [unannounced][topic unannounced channels].  A possible feature
+  that can be added to the proposal will allow nodes to signal to their
+  peers that they want to convert an unannounced channel to a public
+  channel.
+
+- **Draft BIP for sending silent payments with PSBTs:** Andrew Toth
+  [posted][toth sp-psbt] to the Bitcoin-Dev mailing list a draft BIP for
```

Comment:
Follow-up BIP draft today (you're probably aware and it is past the cut-off, but just in case):

- https://github.com/bitcoin/bips/pull/1689
- https://groups.google.com/g/bitcoindev/c/MezoKV5md7s

Reply:
Thanks!  I feel like we've been a bit light on news the past few weeks, so I'm going to save this for next week's newsletter to guarantee that we have something to cover.